Modules/@babylonjs/react-native-windows/windows/CMakeLists.txt

```diff
@@ -53,9 +40,3 @@ target_link_libraries(BabylonNative
     Window
     XMLHttpRequest
     Canvas)
-
-if(DEFINED EXTENSIONS_DIRS)
-    foreach(extensionDir IN LISTS EXTENSIONS_DIRS)
-        add_subdirectory(${extensionDir} "Extensions" EXCLUDE_FROM_ALL)
-    endforeach()
-endif()

```

Comment:
All this is no longer needed or something?

Reply:
We are now adding the top level CMakeLists, so it's already included in that.